ocean_lib/data_provider/test/test_data_service_provider.py

```diff
@@ -161,6 +163,21 @@ def test_delete_job_result(with_nice_client):
     assert result == {"good_job": "with_mock_delete"}
 
 
+def test_encrypt_urls():
+    """Tests successful encrypt URLs job."""
+    file1_dict = {"type": "url", "url": "https://url.com/file1.csv", "method": "GET"}
+    file2_dict = {"type": "url", "url": "https://url.com/file2.csv", "method": "GET"}
+    file1 = FilesTypeFactory(file1_dict)
+    file2 = FilesTypeFactory(file2_dict)
+
+    result = DataSP.encrypt_urls(
+        [file1, file2], "http://localhost:8030/api/services/encrypt"
+    )
+
+    assert isinstance(result, bytes)
+    assert result.decode("utf-8").startswith("0x")
```

Comment:
I don't think this is enough to assert. Can you assert some things about the content itself, or maybe decrypt and check the content is the same?